src/platforms/hosted/stlinkv2.c

```diff
@@ -1099,7 +1099,7 @@ uint32_t stlink_swdp_scan(bmp_info_t *info)
 
 	stlink_dp_error(dp);
 
-	adiv5_dp_init(dp);
+	adiv5_dp_init(dp, UINT32_MAX);
```

Comment:
we can pass 0 here, and the other `adiv5_dp_init` calls alike, since in the end we went for looking for an exact match and not parsing the version from it, that said i dont see anything wrong with passing this either